tests/Dockerfile

```diff
@@ -1,38 +1,32 @@
-FROM rust:latest
+FROM rust
+
+# Install Tarpaulin (code coverage tool)
+# https://github.com/xd009642/tarpaulin
+COPY --from=xd009642/tarpaulin /usr/local/cargo/bin/cargo-tarpaulin /usr/local/cargo/bin/cargo-tarpaulin
 
 # Install Node.js
 ENV NODE_VERSION 12.0.0
-RUN curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash \
```

Comment:
It looked like having `NODE_VERSION` set automatically caused `nvm` to install that version and set it to default. If you check the logs of older runs, we were installing Node multiple times. 😛